conversion/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "babel": {
+    "plugins": ["transform-inline-environment-variables"],
+    "presets": [
+      [
+        "@babel/preset-env",
+        {
+          "targets": "cover 99.5%"
+        }
+      ]
+    ]
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.7.5",
+    "@babel/core": "^7.7.5",
+    "@babel/preset-env": "^7.7.6",
+    "babel-plugin-transform-inline-environment-variables": "^0.4.3"
+  },
+  "scripts": {
+    "build": "babel src/ -d public/"
```

Comment:
Decided to go with just the babel cli for now, webpack is a bit overkill for this imo